api/src/main/java/org/apache/iceberg/types/Comparators.java

```diff
@@ -182,13 +183,39 @@ public int compare(ByteBuffer buf1, ByteBuffer buf2) {
     private CharSeqComparator() {
     }
 
+    private boolean isCharInUTF16HighSurrogateRange(char c1) {
```

Comment:
Here's an example, I think this was removed in a later version. Similarly, the extra newline on line 201 was fixed in another version.